common/compositor/gl/egloffscreencontext.h

```diff
@@ -46,6 +47,7 @@ class EGLOffScreenContext {
   EGLSurface saved_egl_read_ = EGL_NO_SURFACE;
   EGLSurface saved_egl_draw_ = EGL_NO_SURFACE;
   bool restore_context_;
+  std::unique_ptr<OS> os_;
```

Comment:
hmm we shouldnt need this. As I mentioned in last review add an API to NativeDisplay i.e. DisableExplicitSync and during displayinitialization in drmhwctwo, we can call that depending on runtime property.